spec/fixtures/files/metadata_dynamics.xml

```diff
@@ -339,6 +339,78 @@
         <NavigationProperty Name="contact_activity_parties" Type="Collection(mscrm.activityparty)" Partner="partyid_contact"/>
         <NavigationProperty Name="Contact_Letters" Type="Collection(mscrm.letter)" Partner="regardingobjectid_contact_letter"/>
       </EntityType>
+      <EntityType Name="role" BaseType="mscrm.crmbaseentity">
+        <Key>
+          <PropertyRef Name="roleid"/>
+        </Key>
+        <Property Name="importsequencenumber" Type="Edm.Int32"/>
+        <Property Name="componentstate" Type="Edm.Int32"/>
+        <Property Name="solutionid" Type="Edm.Guid"/>
+        <Property Name="_roletemplateid_value" Type="Edm.Guid"/>
+        <Property Name="modifiedon" Type="Edm.DateTimeOffset"/>
+        <Property Name="_modifiedby_value" Type="Edm.Guid"/>
+        <Property Name="_modifiedonbehalfby_value" Type="Edm.Guid"/>
+        <Property Name="organizationid" Type="Edm.Guid"/>
+        <Property Name="name" Type="Edm.String" Unicode="false"/>
+        <Property Name="canbedeleted" Type="mscrm.BooleanManagedProperty"/>
```

Comment:
Here is where the complex type `BooleanManagedProperty` is referenced using the alias instead of the full namespace.